### DISCUSSION TITLE
PC端打不开是怎么回事？

### DISCUSSION THREAD

**Discussion by shuirou520** [xiaohucode/yidaRule#48] [General]
WIN10  打开的时候鼠标转了几个圈 然后就没有然后了 这是怎么回事

**Comment by wufeixiao** [comment]
我win10也打不开，点了没反应

**Reply by xiaohucode** [reply]
把系统版本信息发出来

**Reply by wufeixiao** [reply]
win10的系统只转圈后面就没反应
![1](https://github.com/xiaohucode/yidaRule/assets/105494443/3e0c5688-0d76-487e-9703-3a4454fbe2e7)

**Reply by wufeixiao** [reply]
用另一台win7的提示
![2](https://github.com/xiaohucode/yidaRule/assets/105494443/fdf1026e-bb34-4249-8b4e-3be987b07add)
电脑版本
![4](https://github.com/xiaohucode/yidaRule/assets/105494443/51c3faf2-8ee4-4f27-b3da-72583e8de4e1)

**Reply by xiaohucode** [reply]
> 用另一台win7的提示
> 
> ![2](https://github.com/xiaohucode/yidaRule/assets/105494443/fdf1026e-bb34-4249-8b4e-3be987b07add)
> 
> 电脑版本
> 
> ![4](https://github.com/xiaohucode/yidaRule/assets/105494443/51c3faf2-8ee4-4f27-b3da-72583e8de4e1)
> 
> 

不支持 win7

**Reply by wufeixiao** [reply]
win10是什么原因呢，是什么运行环境没装好吗
